s2clientprotocol/sc2api.proto

```diff
@@ -567,6 +567,8 @@ message Observation {
   optional ObservationUI ui_data = 8;                       // Populated if Feature Layer or Render interface is enabled.
 }
 
+// Only one of these Action fields should be set at once
```

Comment:
Only one of these would be set when making an action, but multiple are likely to be returned when getting the actions back in an observation (eg from a replay) since the action can be represented in multiple ways.

Reply:
Good point!   I just pushed an update